webapp/channels/src/components/drafts/drafts.tsx

```diff
@@ -38,6 +39,29 @@ function Drafts({
 }: Props) {
     const dispatch = useDispatch();
     const {formatMessage} = useIntl();
+    const [activeTab, setActiveTab] = useState<number>();
+
+    const history = useHistory();
+    const match: match<{team: string}> = useRouteMatch();
+    const isDraftsTab = useRouteMatch('/:team/drafts');
+    const isScheduledPostsTab = useRouteMatch('/:team/scheduled_posts');
+
+    useEffect(() => {
+        // sets the active tab based on the URL.
+        // This is in an effect to allow changing tabs when performing
+        // browser back and forward navigations.
+        let tab: number;
+
+        if (isDraftsTab) {
+            tab = 0;
+        } else if (isScheduledPostsTab) {
+            tab = 1;
+        } else {
+            tab = 0;
+        }
+
+        setActiveTab(tab);
```

Comment:
Why not simplifying all this to:

```suggestion
        setActiveTab(isScheduledPostsTab ? 1 : 0);
```
If we ever need more tabs, we can revisit the logic, but we can use a bit of YAGNI here.

Then the effect can just depend on `isScheduledPostsTab` instead of the whole match.

Reply:
Done.